src/main/java/uk/gov/hmcts/ccd/domain/service/FindUserProfileOperation.java

```diff
@@ -18,9 +28,35 @@ public FindUserProfileOperation(UserProfileRepository userProfileRepository) {
     }
 
     public UserProfile execute(String userProfileId, String actionedBy) {
-        final UserProfile userProfile = userProfileRepository.findById(userProfileId, actionedBy);
+        String decodedUid = UriUtils.decode(userProfileId, "UTF-8");
```

Comment:
is this to fix an issue outside the scope? If so, we may need to raise a ticket just to give it a visibility.

Reply:
No, I had just moved the decoding part from the controller to service layer.